### ISSUE TITLE
Full line go template gives an error

### ISSUE THREAD

**Issue by bentatham** [de-jcup/eclipse-yaml-editor#79]
First off - amazing that you added support for Go Templates for Helm in #76 .

If I have a template where the entire file is in a conditional like this:

```
{{ if and .Values.apollo (eq .Values.apollo "true") }}
apiVersion: apps/v1
kind: Deployment
metadata:
  name: apollo
  labels:
    app: apollo
spec:
 ...
{{ end }}
```

I get an error from the yaml editor:

```
mapping values are not allowed here
 in 'reader', line 2, column 11:
    apiVersion: apps/v1
              ^
```

Not sure if it works in all cases, but maybe try to treat all lines that start with `{{` as comments? Or some other hack of the yaml like that?

**Comment by de-jcup** [comment]
Yes you are right:
![image](https://user-images.githubusercontent.com/11644753/68747126-32c5ad00-05fa-11ea-8e17-94f3fe1725ce.png)

To handle those parts as comment should be right choice. Will come with next release.

Thanx for reporting.

**Comment by de-jcup** [comment]
Now correct handled:
![image](https://user-images.githubusercontent.com/11644753/68943360-4b28f980-07ab-11ea-83a5-f5eb9117f43e.png)

![image](https://user-images.githubusercontent.com/11644753/68943406-6bf14f00-07ab-11ea-96ca-a4c265cb33c8.png)

**Comment by de-jcup** [comment]
@bentatham just released 1.6.1 on eclipse marketplace containing the bugfix
